### ISSUE TITLE
Postman test run results in `exec /bin/runner: argument list too long`

### ISSUE THREAD

**Issue by vLia** [kubeshop/testkube#3299]
**Describe the bug**
Issue was encountered by at least 2 users on Discord. 

**To Reproduce**
Command used:
```bash
$ kubectl testkube run test testkubeglobalupload \
--variables-file /home/.../ENV_FileStorage_pm.json \
--upload-timeout 60s \ 
--copy-files "/home/.../temp.txt:temp.txt"
--copy-files "/home/.../temp2.txt:temp2.txt"
--copy-files "/home/.../temp3.txt:temp3.txt"
--copy-files "/home/.../file_250.txt:file_250.txt"
--copy-files "/home/.../image.gif:image.gif"
--copy-files "/home/.../video.mp4:video.mp4"
```
The files cannot be shared due to the sensitive information they contain.
File sizes:

Collection:
![image](https://user-images.githubusercontent.com/10296566/223733575-b87ff535-ce76-4bf4-ad47-5902452d8e82.png)
![image](https://user-images.githubusercontent.com/10296566/223693853-cef0c75c-71a1-4b5a-ac73-1ead42563357.png)

Uploads:
![image](https://user-images.githubusercontent.com/10296566/223694009-9aae52da-a72c-4a1e-ab7a-6d9ebd78e8f0.png)

As the error appears before any validation is done on these files, it should be enough to generate files of these sizes, and pass them to the command in order to reproduce the issue. 
Error: `exec /bin/runner: argument list too long`

**Expected behavior**
With correct files: test run succeeds
With generated files: test run throws error in the test runner container, after the init container

**Version / Cluster**
- Which testkube version? v1.9.21
- What Kubernetes cluster? (e.g. GKE, EKS, Openshift etc, local KinD, local Minikube) ??
- What Kubernetes version? ??

**Screenshots**
If applicable, add CLI commands/output to help explain your problem.
 
**Additional context**

**Comment by sahagnormanian** [comment]
Any updates?

**Comment by vLia** [comment]
Managed to reproduce the error on mac with a large environment file:
`standard_init_linux.go:228: exec user process caused: argument list too long`

Copy-files are sent in by their file name, which is significantly shorter. Therefore I will look into options on how to handle big environment files.

**Comment by vLia** [comment]
Hello @sahagnormanian, 
A fix was released with Testkube v1.11.22. Could you please check if it solved this for you and let us know? TIA

**Comment by TheBrunoLopes** [comment]
Hello @sahagnormanian , seems like this issue was fixed.
Let us know if it's not the case and we will re-open it ;)
